.snippets/code/application-devs/interact/sudo-metadata-calls.json

```diff
@@ -0,0 +1,84 @@
+{
+    "id": 117,
+    "type": {
+        "path": ["pallet_sudo", "pallet", "Call"],
+        "params": [
+            {
+                "name": "T",
+                "type": null
+            }
+        ],
+        "def": {
+            "variant": {
```

Comment:
When using Polkadot API's observable client, metadata is formatted differently than this raw format, and this may cause confusion. 

I have been using the `client.chainHead$().metadata$` subscription from https://github.com/polkadot-api/polkadot-api/tree/main/packages/observable-client to retreive metadata using PAPI. Perhaps it is worth noting somewhere that APIs may format Metadata differently to the raw format? 

I am not aware of metadata plans moving forward beyond some public commentary. But for pallets especially a tag convention is being used with observable client, such as a `def.tag` field of `variant` as its value, instead of `def.variants.variant`. And `def.value.fields` instead of `def.variants.variant.fields`.

This results in developer tools having to adapt their scraping to the different format. And I imagine there is a push for teams to use Polkadot API on the application side.

Reply:
I see; for now, will add a note regarding the format, but maybe in the future we can amend / elaborate on this a little more